utils/src/test/kotlin/storage/FileArchiverTest.kt

```diff
@@ -124,5 +124,22 @@ class FileArchiverTest : StringSpec() {
                 assertFileContent("path/LICENSE")
             }
         }
+
+        "The pattern matching is case-insensitive" {
+            createFile("LICENSE")
+            createFile("License")
+            createFile("license")
+            createFile("LiCeNsE")
+
+            FileArchiver.DEFAULT.archive(workingDir, "save")
+            FileArchiver.DEFAULT_ARCHIVE_DIR.resolve("save/archive.zip").unpack(targetDir)
```

Comment:
This should use a temporary directory instead.

Reply:
Done.